Document container-based primary tags as enabled by default (Agent 7.65.0) (#38381)
DataDog/documentation: content/en/tracing/guide/setting_primary_tags_to_scope.md

## Container-based additional primary tags

You can index your trace metrics based on the tags derived from Docker containers and Kubernetes pod metadata on Linux-based platforms.

> Container-based primary tags are enabled by default in Datadog Agent versions 7.65.0 and later. In versions 7.35.0 through 7.64.x, the feature is opt-in outside of Fargate and enabled automatically in Fargate environments.

Reviewer: Optional clarity tweak — the Fargate exception is stated in both directions, which reads a little dense.

```suggestion
Container-based primary tags are enabled by default in Datadog Agent versions 7.65.0 and later. In Agent versions 7.35.0 through 7.64.x, container-based primary tags are enabled automatically in Fargate and are opt-in everywhere else.
```

With Agent 7.65.0 or later, no feature flag is required. Go to the [APM Settings][6] page and select the additional primary tag you want to use. It can take up to two hours for changes to this setting to take effect.

> Now you can filter your services in the [Catalog][7] by the tag being sent by your containerized services. Trace metrics used by Dashboards and Monitors can also be aggregated by the container primary tag.

Reviewer: Minor — Vale flags "Now" as a temporal word (`Datadog.tense`). Pre-existing text just relocated by this PR, so not blocking, but since this hunk is already touched:

```suggestion
You can filter your services in the [Catalog][7] by the tag being sent by your containerized services. Trace metrics used by Dashboards and Monitors can also be aggregated by the container primary tag.
```

**Note**: Primary tag values should not contain capital letters or special characters (aside from underscores, minuses, colons, periods, and slashes). If they do, some features may not work properly.

### Disable container-based primary tags

To turn off container-based primary tags, set the `disable_cid_stats` APM feature and restart the Agent. If `DD_APM_FEATURES` is already set, append `disable_cid_stats` to its comma-separated list. The procedure depends on how you installed the Agent:

{{< tabs >}}
{{% tab "Helm" %}}

> Add the following to your values file:

Reviewer: The removed sentence dropped the minimum Helm chart version needed to set `DD_APM_FEATURES` (previously: "Using the Datadog Helm chart version 2.26.2 or later, add the following to your values file:"). Can you confirm this constraint no longer applies (e.g., that chart version is old enough it's a non-issue), or should it be kept for the disable flow?

Author: That version is [4.5 years old](https://github.com/DataDog/helm-charts/releases/tag/datadog-2.26.2), we can safely drop it from the sentence IMO.
